### PR TITLE
Fix travis errors that are occurring due to ember-cli-sauce config; Bumped node version to 8.x+ and therefore bumped package version to 1.x

### PR DESCRIPTION
**Pull request by evanfarina**
Recent commits have been failing travis runs due to a slew of issues originating from ember-cli-sauce. Some of these issues include:

- An outdated version of sauce is used if not sauce_version is explicitly listed in travis.yml. An error regarding an old version of sauce being used can be seen in many of travis logs of recent commits to this repo.
- Some of the args being passed were not being recognized by ember-cli-sauce. Doesn't help that their docs are wildly out of date :) Moving away from the shorthand syntax for a few of the args solves this issue.
- Executing the ember-cli-sauce executable directly
- Tested by downloading and configuring sauce locally and running against my local sauce config via yarn test:sauce.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-asset-loader",
-  "version": "0.6.1",
+  "version": "1.0.0",
```
Comment (rwjblue) [inline package.json]: Please revert this change, generally the version number bumping is a "release time" thing (and not done in a PR itself).

Comment (evanfarina) [reply package.json]: Got it, thanks for letting me know.

**Review by rwjblue — APPROVED** [review]
